.github/workflows/standard-workflow.yml

```diff
@@ -41,6 +41,10 @@ jobs:
         run: sbt scalafmtCheckAll scalafmtSbtCheck "scalafixAll --check"
         working-directory: examples/akka-cluster-app
 
+      - name: Check code style - examples/akka-persistence-app
+        run: sbt scalafmtCheckAll scalafmtSbtCheck "scalafixAll --check"
```

Comment:
Actually... you can use YAML anchor here, see https://github.com/VirtusLab/git-machete-intellij-plugin/blob/develop/.circleci/config.yml#L222

Reply:
Are you sure this is not specific for circleci?

Reply:
https://github.com/actions/runner/issues/1182 - looks like YAML anchors are not available for .github workflows yet?